stl/src/atomic.cpp

```diff
@@ -8,6 +8,14 @@
 #include <intrin.h>
 #pragma warning(disable : 4793)
 
+#include "atomic"
+#include "awint.h"
+#include "cstdint"
+#include "new"
+#include "thread"

```

Comment:
We use angle-bracket includes for headers intended to be found in the include path, and double-quote includes for pathnames relative to the source file.
```suggestion
#include <atomic>
#include "awint.h"
#include <cstdint>
#include <new>
#include <thread>
```